tests/packages/test_file_dependency.py

```diff
@@ -62,3 +65,31 @@ def test_file_dependency_pep_508_local_file_relative_path(mocker):
 
     requirement = "{} @ {}".format("demo", path)
     _test_file_dependency_pep_508(mocker, "demo", path, requirement)
+
+
+def test_to_pep_508_with_marker(mocker):
```

Comment:
I think we can use a parameterize test here, to test absolute and relative path.